RELEASE-NOTES.md

```diff
@@ -1,12 +1,14 @@
 # Wikibase DataModel Serialization release notes
 
-## 2.5.0 (dev)
+## 2.5.0 (2017-08-30)
 
+* Removed MediaWiki integration files.
+* Added compatibility with DataValues Geo 2.x.
 * Deprecated `SerializerFactory` options
   `OPTION_SERIALIZE_MAIN_SNAKS_WITHOUT_HASH`,
   `OPTION_SERIALIZE_QUALIFIER_SNAKS_WITHOUT_HASH` and
   `OPTION_SERIALIZE_REFERENCE_SNAKS_WITHOUT_HASH`,
-  and added `OPTION_SERIALIZE_SNAKS_WITHOUT_HASH` instead
+  and added `OPTION_SERIALIZE_SNAKS_WITHOUT_HASH` instead.
```

Comment:
I agree the period looks nicer, but the older release notes all have list items without a period at the end…